packages/api/src/auth/github/github.client.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@nestjs/common';
+import { PassportStrategy as GithubStrategy } from '@nestjs/passport';
+import { Strategy } from 'passport-github2';
+import { VerifiedCallback } from 'passport-jwt';
+import type { UserDTO } from 'src/users/users.model';
+
+import { UsersMapper } from '../../users/users.mapper';
+import { UsersRepository } from '../../users/users.repository';
+import { JwtStrategy } from '../jwtStrategy/jwt.strategy';
+import type { GithubResponse, GithubUserData } from './github.model';
+
+@Injectable()
+export class GithubClient extends GithubStrategy(Strategy) {
+  constructor(private usersRepository: UsersRepository, private jwtStrategy: JwtStrategy) {
+    super({
+      clientID: process.env.GITHUB_CLIENT_ID,
```

Comment:
Use environment variables from `env.ts` as they would be validated and type-safe.